auxx/src/Lang/Lexer.hs

```diff
@@ -57,6 +57,16 @@ import           Pos.Util.Util (toParsecError)
 
 import           Test.Pos.Core.Arbitrary ()
 
+import           Formatting.Buildable   (Buildable (build))
+import           Data.Text.Lazy.Builder (toLazyText)
+import           Data.Text.Lazy         (toStrict)
+----------------------------------------------------------------------------
+-- Compat shims
+----------------------------------------------------------------------------
+-- pretty used to be in Universum
+pretty :: Buildable a => a -> Text
+pretty = toStrict . toLazyText . build
```

Comment:
@parsonsmatt I didn't know how often this would come up initially. It ended up being used quite a bit.

Ideally, I guess we had our own fork of `universum`, and just put it back in?

Reply:
I'd be a fan of forking `univeresum` directly into this repository, tbh.

Reply:
maybe `Pos.Util.Util` could host this "pretty" function?

Reply:
I guess what we are *supposed* to use is `fmt` from `fmt` instead. I'll update as needed.

Will validate that the space leak is gone first though and then drop the `pretty` redefinitions.

Reply:
> I'd be a fan of forking univeresum directly into this repository, tbh.

Oh, please no! The cardano-sl repo is already too big and I'd really like to split it up. I really need to write a document about this explain why and how and the benefits of doing this.